### ISSUE TITLE
The <template> element is deprecated.

### ISSUE THREAD

**Issue by nischi** [valor-software/ngx-bootstrap#2007]
```
[x] bug report
[ ] support request
[ ] feature request
```

**Current behavior**
it works, but if i compile the application with AOT they gave me a warning 

```
The <template> element is deprecated. Use <ng-template> instead ("
[WARNING ->]<template [ngIf]="!isClosed">
  <div [class]="'alert alert-' + type" role="alert" [ngClass]="classes""): ng:///C:/..../node_modules/ngx-bootstrap/alert/alert.component.d.ts.AlertComponent.html@1:0
```

**Expected/desired behavior**
No Warning

**Reproduction of the problem**
Compile the application where you use alerts with AOT compiler.

**What is the motivation / use case for changing the behavior?**
No Warnings in my compile :)

**Please tell us about your environment:**
* **ngx-bootstrap version:** 1.6.6
* **Angular version:** 4.1.3

**Comment by Martin-Luft** [comment]
Please search for same issues before opening a new one...

For Angular 4 just use the @next version which fixes this warning. This cannot be fixed in 1.6.X because it would break Angular 2 support. You can use "ngx-bootstrap": "2.0.0-beta.2" or npm install ngx-bootstrap@next

**Comment by nischi** [comment]
Sorry about that, i have searched in the issues, but not found anything and not realized that a version 2 is in beta now. wil download this, thanks.
